### PR TITLE
[FDD] QA task to check the pilot beam data

### PR DESCRIPTION
**Pull request by arvindkhuntia**

### CODE REVIEW

**Review by iarsene — APPROVED** [review]

**Comment by iarsene** [comment]
@jgrosseo  Sorry, i missed your comment. Yes, i agree.

**Comment by jgrosseo** [comment]
for DPG It should be @mario-krueger who approves...

**Review by mario-krueger — APPROVED** [review]

File: DPG/Tasks/CMakeLists.txt [hunk 1]
```diff
@@ -10,50 +10,54 @@
...
 o2physics_add_dpl_workflow(pid-tpc-qa-mc
-                    SOURCES qaPIDTPCMC.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
-                    COMPONENT_NAME Analysis)
+        SOURCES qaPIDTPCMC.cxx
+        PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
+        COMPONENT_NAME Analysis)
+##FDD
+  o2physics_add_dpl_workflow(fdd-qa
+          SOURCES fddQA.cxx
```
Comment (mario-krueger) [inline DPG/Tasks/CMakeLists.txt:62]: you could consider renaming the task to ´qaFDD´ to be consistent with the other DPG-QA task names.

**Comment by jgrosseo** [comment]
When you update, you can also remove the CMakeLists.txt in Common from this commit (the spaces can be changed by someone else)

**Review by mario-krueger — APPROVED** [review]

**Pull request merged**